src/main/java/teammates/ui/template/FeedbackResponseCommentRow.java

```diff
@@ -308,8 +308,8 @@ private String getTypeOfPeopleCanViewComment(FeedbackResponseCommentAttributes c
             case RECEIVER:
                 peopleCanView.append("response recipient, ");
                 break;
-            case OWN_TEAM:
```

Comment:
The situation get deleted because?

Reply:
```
public enum FeedbackParticipantType {
    // booleans represent: isValidGiver?, isValidRecipient? isValidViewer?
    // Strings represents: option shown in giver select box, option shown in recipient select box,
    // text displayed during feedback submission respectively.
    SELF(true, true, false, "Feedback session creator (i.e., me)", "Giver (Self feedback)", ""),
    STUDENTS(true, true, true, "Students in this course", "Other students in the course", "Other students in the course"),
    //used to generate options for MCQ & MSQ:
    STUDENTS_EXCLUDING_SELF(false, false, false, "Students in this course", "Other students in the course",
            "Other students in the course"),
    INSTRUCTORS(true, true, true, "Instructors in this course", "Instructors in the course", "Instructors in this course"),
    TEAMS(true, true, false, "Teams in this course", "Other teams in the course", ""),
    TEAMS_EXCLUDING_SELF(false, false, false, "Teams in this course", "Other teams in the course", ""),
    OWN_TEAM(false, true, false, "", "Giver's team", "Your team"),
    OWN_TEAM_MEMBERS(false, true, true, "", "Giver's team members", "Your team members"),
    OWN_TEAM_MEMBERS_INCLUDING_SELF(false, true, true, "", "Giver's team members and Giver", "Your team members"),
    RECEIVER(false, false, true, "", "", "The receiving"),
    RECEIVER_TEAM_MEMBERS(false, false, true, "", "", "The recipient's team members"),
    NONE(false, true, false, "", "Nobody specific (For general class feedback)", ""),
    // Used by feedbackResponseComment:
    GIVER(false, false, true, "", "", "");
```

From here we can see that `OWN_TEAM` is not a valid viewer.